deployment/ansible/roles/ckan-odp-configuration/templates/nginx_ckan.j2

```diff
@@ -23,4 +23,16 @@ server {
         # In emergency comment out line to force caching
         # proxy_ignore_headers X-Accel-Expires Expires Cache-Control;
     }
+    location = / {
+        proxy_pass http://127.0.0.1:8080/;
+        proxy_set_header Host $host;
+        proxy_cache cache;
+        proxy_cache_bypass $cookie_auth_tkt;
```

Comment:
This was actually in the default CKAN config, so there wasn't much to figure out.